tests/testthat/test-users.R

```diff
@@ -13,5 +13,5 @@ test_that("lookup_users works", {
   usr_df <- lookup_users(users)
   expect_s3_class(usr_df, "tbl")
   expect_equal(nrow(usr_df), 6)
-  expect_equal(ncol(usr_df), 91)
+  expect_equal(ncol(usr_df), 20)
```

Comment:
So now lookup_users will return less data than previously? Wouldn't make more sense to add more data to what search_users instead?

Reply:
I don't think so — I think `lookup_users()` now returns more appropriate data. 

Before

```
 > lookup_users("hadleywickham")
# A tibble: 1 x 91
  user_id  status_id   created_at          screen_name  text             source  display_text_wi…
  <chr>    <chr>       <dttm>              <chr>        <chr>            <chr>              <int>
1 69133574 1379045152… 2021-04-05 12:16:00 hadleywickh… @scheidegger I’… Twitte…               NA
# … with 84 more variables: reply_to_status_id <chr>, reply_to_user_id <chr>,
#   reply_to_screen_name <chr>, is_quote <lgl>, is_retweet <lgl>, favorite_count <int>,
#   retweet_count <int>, quote_count <int>, reply_count <int>, hashtags <list>, symbols <list>,
#   urls_url <list>, urls_t.co <list>, urls_expanded_url <list>, media_url <list>,
#   media_t.co <list>, media_expanded_url <list>, media_type <list>, ext_media_url <list>,
#   ext_media_t.co <list>, ext_media_expanded_url <list>, ext_media_type <chr>,
#   ext_alt_text <list>, mentions_user_id <list>, mentions_screen_name <list>, lang <chr>,
#   quoted_status_id <chr>, quoted_text <chr>, quoted_created_at <dttm>, quoted_source <chr>,
#   quoted_favorite_count <int>, quoted_retweet_count <int>, quoted_user_id <chr>,
#   quoted_screen_name <chr>, quoted_name <chr>, quoted_followers_count <int>,
#   quoted_friends_count <int>, quoted_statuses_count <int>, quoted_location <chr>,
#   quoted_description <chr>, quoted_verified <lgl>, retweet_status_id <chr>,
#   retweet_text <chr>, retweet_created_at <dttm>, retweet_source <chr>,
#   retweet_favorite_count <int>, retweet_retweet_count <int>, retweet_user_id <chr>,
#   retweet_screen_name <chr>, retweet_name <chr>, retweet_followers_count <int>,
#   retweet_friends_count <int>, retweet_statuses_count <int>, retweet_location <chr>,
#   retweet_description <chr>, retweet_verified <lgl>, place_url <chr>, place_name <chr>,
#   place_full_name <chr>, place_type <chr>, country <chr>, country_code <chr>,
#   geo_coords <list>, coords_coords <list>, bbox_coords <list>, status_url <chr>, name <chr>,
#   location <chr>, description <chr>, url <chr>, protected <lgl>, followers_count <int>,
#   friends_count <int>, listed_count <int>, statuses_count <int>, favourites_count <int>,
#   account_created_at <dttm>, verified <lgl>, profile_url <chr>, profile_expanded_url <chr>,
#   account_lang <lgl>, profile_banner_url <chr>, profile_background_url <chr>,
#   profile_image_url <chr>
```

After:

```
> lookup_users("hadleywickham")
# A tibble: 1 x 20
  user_id  name   screen_name location description  url   protected followers_count friends_count
* <chr>    <chr>  <chr>       <chr>    <chr>        <chr> <lgl>               <int>         <int>
1 69133574 Hadle… hadleywick… Houston… R, data, vi… http… FALSE              119565           270
# … with 11 more variables: listed_count <int>, statuses_count <int>, favourites_count <int>,
#   account_created_at <dttm>, verified <lgl>, profile_url <chr>, profile_expanded_url <chr>,
#   account_lang <lgl>, profile_banner_url <chr>, profile_background_url <chr>,
#   profile_image_url <chr>
```

If you really do want the tweet data you can still get at it with `tweets_data()`.

Reply:
Sorry, I misunderstood the impact of having `tweet_data`.